app/assets/javascripts/deploys.js

```diff
@@ -126,43 +129,47 @@ $(function () {
 
     $messages.scrollTop($messages.prop("scrollHeight"));
 
-    $("#output-options > button, #output-grow-toggle").removeClass("active");
+    $("#output-options > button, #output-expand-toggle").removeClass("active");
     $(this).addClass("active");
   });
 
-  function growOutput() {
-    $messages.css("max-height", "none");
-  }
-
-  $("#output-grow-toggle").click(function() {
-    var $self = $(this);
-
-    if($self.hasClass("active")) {
-      shrinkOutput();
-      $self.removeClass("active");
-    } else {
-      growOutput();
-      $self.addClass("active");
-    }
-  });
+  $("#output-no-follow").click(function() {
+    following = false;
 
-  $("#output-grow").click(function() {
-    growOutput();
+    shrinkOutput();
 
     $("#output-options > button").removeClass("active");
```

Comment:
or if they're mutually exclusive then we should have this line in the other handler too?